js/app/modules/selection/xapian.js

```diff
@@ -109,6 +109,11 @@ const Xapian = new Module.Class({
     clear: function () {
         this._get_more = null;
         this._query_index = 0;
+        let reset_can_load_more = true;
+        if (reset_can_load_more !== this._can_load_more) {
+            this._can_load_more = reset_can_load_more;
+            this.notify('can-load-more');
+        }
         this._can_load_more = true;
```

Comment:
This statement is a little bit confusing, what does "reset_can_load_more"? Does it determine whether we want to reset the value or not, or is it representing the original value? In any case, do we really need to assign the same value twice?